src/components/CatFilters.jsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+
+export const CatFilters = ({ revList, innerUrl }) => {
+  const [filteredRev, setFilteredRev] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [activeLink, setActiveLink] = useState(false);
+
+  useEffect(() => {
+    setFilteredRev((currRev) => {
+      setLoading(true);
+      const newList = revList.filter((rev) => {
```

Comment:
This needs to be done using request queries, have a look at how your backend can filter getReviews based on this.

Reply:
my backend does not filter based on category